examples/python/helloworld/greeter_client.py

```diff
@@ -26,6 +26,7 @@ def run():
     # NOTE(gRPC Python Team): .close() is possible on a channel and should be
     # used in circumstances in which the with statement does not fit the needs
     # of the code.
+    print("Will try to greet world ...")
```

Comment:
This is a little bit cryptic. What value do you think this log adds?

Reply:
I'm using the same wording in [Java quick start](https://grpc.io/docs/languages/java/quickstart/#run-the-example), this indicates the client is up and running and about to start the actual RPC call.

Reply:
Fair enough. Matching other implementations is a decent argument.